prompt_toolkit/output/vt100.py

```diff
@@ -683,3 +683,6 @@ def bell(self) -> None:
         " Sound bell. "
         self.write_raw("\a")
         self.flush()
+
+    def get_default_color_depth(self) -> ColorDepth:
+        return ColorDepth.vt100_default(self.term)
```

Comment:
I would actually move the implementation of ``vt100_default`` in here. (and same for the other Output implementations.)

Basically the open/closed principle: no need to change the ``ColorDepth`` class if we define a new ``Output``.
I can make that change.

Reply:
That makes sense! I tried to keep the code for choosing a color depth at the same place to make it clear which logic is used in which case.